integrations/pjrt/README.md

```diff
@@ -78,14 +77,7 @@ The plugin `openxla_pjrt_artifacts` is in the `ctstools` directory and
 performs additional manipulation of the environment in order to save
 compilation artifacts, reproducers, etc.
 
-## Contacts
-
-* [GitHub issues](https://github.com/openxla/openxla-pjrt-plugin/issues):
-  Feature requests, bugs, and other work tracking
-* [OpenXLA discord](https://discord.gg/pvuUmVQa): Daily development discussions
-  with the core team and collaborators
```

Comment:
Now an invalid invite link. Also seems the OpenXLA community no longer maintains it.

Reply:
Could link to https://github.com/openxla/community?tab=readme-ov-file#communication-channels. The OpenXLA Discord server still exists but it isn't very active.

Reply:
Done : )